packages/codemod/transformations/core/v2-to-v3/attention-box-component-migration.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  findComponentElements,
+  getComponentNameOrAliasFromImports,
+  getCoreImportsForFile,
+  updatePropName,
+  wrap
+} from "../../../src/utils";
+import { TransformationContext } from "../../../types";
+
+/**
+ * 1. Update the 'componentClassName' prop to 'className'
+ */
+function transform({ j, root }: TransformationContext) {
+  const imports = getCoreImportsForFile(root);
+  const actualComponentName = getComponentNameOrAliasFromImports(j, imports, "AttentionBox");
```

Comment:
Im not sure what i think about the name "actualComponentName" (since it is not the actual, it is the alias) how about just "componentName" or "importedComponentName"?